dining_guide.rst

```diff
@@ -6,7 +6,7 @@ Seattle Dining Guide
 .. contents::
    :local:
 
-U-District (Conference Vicinity)
+U-District (Conference Vicinity Lunch)
 --------------------------------
```

Comment:
```suggestion
--------------------------------------
```

Reply:
Just need to make the dashes same length as the previous line, so Travis will pass :)

Reply:
Sounds good. Thank you.